src/include/duckdb.h

```diff
@@ -3759,6 +3933,21 @@ Append a duckdb_timestamp value to the appender.
 */
 DUCKDB_API duckdb_state duckdb_append_timestamp(duckdb_appender appender, duckdb_timestamp value);
 
+/*!
+Append a duckdb_timestamp_s value to the appender.
+*/
+DUCKDB_API duckdb_state duckdb_append_timestamp_s(duckdb_appender appender, duckdb_timestamp_s value);
+
+/*!
+Append a duckdb_timestamp_ms value to the appender.
+*/
+DUCKDB_API duckdb_state duckdb_append_timestamp_ms(duckdb_appender appender, duckdb_timestamp_ms value);
+
+/*!
+Append a duckdb_timestamp_ns value to the appender.
+*/
+DUCKDB_API duckdb_state duckdb_append_timestamp_ns(duckdb_appender appender, duckdb_timestamp_ns value);
```

Comment:
Can we leave these out as well? The `duckdb_append_` methods are not intended to be complete. You can use `duckdb_append_data_chunk` to append data chunks containing any type within DuckDB (and we should add `duckdb_append_value` at some point to append a `duckdb_value`).